linux_os/guide/system/bootloader-grub2/grub2_password/rule.yml

```diff
@@ -8,28 +8,15 @@ description: |-
     The grub2 boot loader should have a superuser account and password
     protection enabled to protect boot-time settings.
     <br /><br />
-    To do so, select a superuser account name and password and and modify the
-    <tt>/etc/grub.d/01_users</tt> configuration file with the new account name.
-    <br /><br />
```

Comment:
Why would this text be removed? Is it no longer accurate?

Reply:
The rule has been split, and this modification has been moved to the other rule.